src/common/Defs.h

```diff
@@ -31,28 +31,34 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef COMMON_DEFS_H_
 #define COMMON_DEFS_H_
 
-#define PRODUCT_NAME        "Unvanquished"
+
+// GAME INFO
+// UNVANQUISHED
+#define PRODUCT_NAME_UNV        "Unvanquished"
 /** Case, No spaces */
-#define PRODUCT_NAME_UPPER  "UNVANQUISHED"
+#define PRODUCT_NAME_UPPER_UNV  "UNVANQUISHED"
 /** No case, No spaces */
-#define PRODUCT_NAME_LOWER  "unvanquished"
+#define PRODUCT_NAME_LOWER_UNV  "unvanquished"
 
-#define PRODUCT_VERSION     "0.50"
+#define PRODUCT_VERSION_UNV     "0.50"
 
 /** Default base package */
-#define DEFAULT_BASE_PAK    PRODUCT_NAME_LOWER
+#define DEFAULT_BASE_PAK_UNV    PRODUCT_NAME_LOWER_UNV
 
 /** URI scheme and server browser filter */
-#define GAMENAME_STRING     "unv"
-#define GAMENAME_FOR_MASTER PRODUCT_NAME_UPPER
+#define GAMENAME_STRING_UNV     "unv"
+#define GAMENAME_FOR_MASTER_UNV PRODUCT_NAME_UPPER_UNV
+
+#define MASTER_SERVER_NAME_UNV  "master.unvanquished.net"
 
-#define MASTER_SERVER_NAME  "master.unvanquished.net"
+#define IRC_SERVER_UNV          "irc.freenode.org"
+#define IRC_CHANNEL_UNV         "unv-lobby"
 
-#define IRC_SERVER          "irc.freenode.org"
-#define IRC_CHANNEL         "unv-lobby"
+#define WWW_BASEURL_UNV         "dl.unvanquished.net/pkg"
```

Comment:
I feel an `UNV`-suffixed definition or Cvar is still a branding that shouldn't be present in a game-agnostic engine.